go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/TomOnTime/utfutil v0.0.0-20180511104225-09c41003ee1d
 	github.com/alicebob/miniredis v2.5.0+incompatible
 	github.com/alicebob/miniredis/v2 v2.14.2
-	github.com/argoproj/gitops-engine v0.7.1-0.20220804190909-2bc3fef13e07
+	github.com/argoproj/gitops-engine v0.7.1-0.20220819195816-9970faba8138
```

Comment:
I think you need to include gitops change https://github.com/argoproj/gitops-engine/commit/a56a8030319ee3af0f6287865133403f8f9ad380 to fix integration test.

Reply:
@ashutosh16 actually it is the opposite. I updated gitops-engine with current master which already includes that commit. The problem is that, the signature change in `gitopsengine.sync.WithNamespaceCreation` breaks current ArgoCD master. Basically current gitops-engine in master is incompatible with current ArgoCD in master.

@pasha-codefresh @jannfis What would be the best course of action in this case? Maybe we should think about how we can prevent this from happening in the future? WDYT?